src/Controller/Component/PropertiesComponent.php

```diff
@@ -95,9 +100,10 @@ public function startup(): void
 
             return;
         }
-
+        /** @phpstan-ignore-next-line */
+        $this->Config->read('Properties');
```

Comment:
I suggest this change to avoid too many cache hits and to avoid changes in `Control` and `PropertyHelper` classes 
In `ConfigComponent::read()` line 71 we could write: 
```php
        if (!empty($config)) {
            $content = (array)json_decode((string)Hash::get($config, 'attributes.content'), true);
            Configure::write($key, $content);
        }
``` 

This way here in `PropertiesComponent` we would have:

```php
Configure::load('properties');
$properties = $this->Config->read('Properties');
```

`Control` and `PropertyHelper` will keep using `Configure::read` and remain unchanged